plugins/generic/googleScholar/GoogleScholarPlugin.inc.php

```diff
@@ -22,7 +22,7 @@ class GoogleScholarPlugin extends GenericPlugin {
 	function register($category, $path, $mainContextId = null) {
 		if (parent::register($category, $path, $mainContextId)) {
 			if ($this->getEnabled($mainContextId)) {
-				HookRegistry::register('CatalogBookHandler::view', array($this, 'monographFileView'));
```

Comment:
This leaves `monographFileView` unused, right? If that function really isn't necessary, could you remove it?

Reply:
@asmecher 
I have removed the unncesary monographFileView method with the following commit.
https://github.com/pkp/omp/pull/915/commits/7cf5ab4dda857992d12dacf7bb5657cddc4ada52